### PR TITLE
Doc for `get_or_set` [ci skip]

### PR DESCRIPTION
**Pull request by ecoologic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,10 @@ cache.set("key", "value")
 puts(cache.get("key")) # "value"
 sleep 30
 puts(cache.get("key")) # nil
+cache.get_or_set("key" { 1 } # 1
+cache.get_or_set("key" { 2 } # 1 (cached value is returned, block is not executed)
```
Comment (leighmcculloch) [inline README.md]: The `get_or_set` function call is missing a closing parenthesis.

**Review by leighmcculloch — CHANGES_REQUESTED** [review]
One thing that needs fixing, otherwise 👍 .

**Comment by ecoologic** [comment]
Oops... good catch - corrected.
@leighmcculloch

**Review by leighmcculloch — APPROVED** [review]

**Comment by leighmcculloch** [comment]
Thanks @ecoologic!

**Pull request merged**